polaris-react/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -18,23 +24,25 @@ interface State {
   };
 }
 
-interface Color extends HSBColor {
```

Comment:
I'm worried that removing `extends HSBColor` might be a breaking change. I don't have enough context around this component to say with certainty though

Reply:
It was not removed, just [moved to the child](https://github.com/Shopify/polaris/pull/7041/files#diff-661878f206f55c65efaab4a68cc91a811fbef957f55dc0820835e672694a2960R15) and  [extended from its interface](https://github.com/Shopify/polaris/pull/7041/files#diff-ed174c9120bd9ba2bd49ee23c72483e55d7cbbc1ec9ffceaf9f300aee336f59fR41).

Doing the other way around would create cycle dependency